build-spring.gradle

```diff
@@ -15,13 +15,23 @@ subprojects {
 
     logger.info("found spring boot relevant project:$project")
     
-	apply plugin: 'io.spring.dependency-management'
+    if (projectType.bootableSpringApplicationProjects.contains(project)){
+       apply plugin: 'org.springframework.boot' 
+       
+       springBoot {
+             // createds build-info.properties in META-INF folder which is auto injected in spring
```

Comment:
I don't understand this comment. Does this mean there should be a property file build-info.properties within the META-INF folder of the jar?

Because I cant't find such a properties file.

Reply:
Means : next step 'buildInfo()' does configure gradle spring boot plugin to generate automatically the META-INF/build-info-properties file when a java compile is running. And yes the file was not created before.

The gradle build does automatically inspect git tags and checks for `v$major.$minor.$hotfix-server` tags. This information is given to gradle and so generated to the properties file.

I tested the behaviour as follows:
- checked out master branch (has tag v0.19.1)
- called `./gradlew clean bootJar` and tried out DAUI check version and got not the 0.19.1
- made my changes in build-spring.gradle
- called `./gradlew clean bootJar` again and tried out DAUI check version and got then the 0.19.1 (as wished)
- then I switched to develop, created new branch and then committed + pushed to git repository

To test it again you could temporary tag your checked out branch, e.g. `git tag v0.19.2-server` and then do the `./gradlew clean bootJar` operation. After this your DAUI check version call must return 0.19.2 as version...

Reply:
👍 I tried it. Works!